src/styl/fonts/fonts.styl

```diff
@@ -0,0 +1,89 @@
+/* This stylesheet generated by Transfonter (https://transfonter.org) on August 30, 2017 7:40 PM */
+
+// Druk
+
+@font-face {
+	font-family: 'Druk';
+	src: url('../static/fonts/Druk-Bold.eot');
+	src: url('../static/fonts/Druk-Bold.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Druk-Bold.woff') format('woff'),
+		url('../static/fonts/Druk-Bold.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
+}
+
+// Post Grotesk
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-Medium.eot');
+	src: url('../static/fonts/PostGrotesk-Medium.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-Medium.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-Medium.ttf') format('truetype');
+	font-weight: bold;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-Regular.eot');
+	src: url('../static/fonts/PostGrotesk-Regular.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-Regular.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-Regular.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot');
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-BookItalic.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-BookItalic.ttf') format('truetype');
+	font-weight: normal;
+	font-style: italic;
```

Comment:
Book Italic not Regular Italic?

Reply:
yea its actually regular italic. the file name was wrong